### ISSUE TITLE
[Native] Implement hamming_distance()

### ISSUE THREAD

**Issue by yzhang1991** [prestodb/presto#20531]
https://github.ibm.com/lakehouse/velox/pull/21
External: https://github.com/facebookincubator/velox/pull/6120

**Comment by amitkdutta** [comment]
@yzhang1991 This PR looks like inside a velox fork inside IBM. Are we planning to put it in https://github.com/facebookincubator/velox? CC: @aditi-pandit @mbasmanova

**Comment by aditi-pandit** [comment]
@amitkdutta : Yes, this PR will be moved to the public repo. I think Arun wanted to get some internal feedback before opening in the public repo.

**Comment by wills-feng** [comment]
I'm taking over this one, new PR is https://github.com/facebookincubator/velox/pull/9269

**Comment by wills-feng** [comment]
PR got merged.

**Comment by aditi-pandit** [comment]
@wills-feng : Please add Prestissimo e2e tests for this function. You can add a test in https://github.com/prestodb/presto/blob/master/presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

**Comment by wills-feng** [comment]
@aditi-pandit thank you! Will follow up.
